Trio.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,5 +1,5 @@
 {
-  "originHash" : "89074a88ed67a58ecd7534519854c5a0928a4046d7c8a6123a7d70f27bf8b44d",
+  "originHash" : "19d4832d449c850d10dc8b15597db0825e41ce3f2a3b0d4b8471c7f66a7517dc",
```

Comment:
Can you remove the changes to this file? Xcode will automatically change it, but in this change it looks like it removed tidepoolKit.